scratchconnect/CloudConnection.py

```diff
@@ -146,7 +155,7 @@ def set_cloud_variable(self, variable_name, value):
             }
             self._send_packet(packet)
             return True
-        except ConnectionAbortedError or BrokenPipeError:

```

Comment:
You have removed an exception. Are you sure it is fixed permanently?

Reply:
It was my problem when I first created this library. I wanted to update that exception code but I kept forgetting it. I can also add a tuple and all the exceptions in it which I want to catch because in python you can't catch exceptions seperated by "or" with the "except" keyword.

I have the beta version 3.1 in my local computer in which I made those changes...

Reply:
Oh ok

Reply:
> You have removed an exception. Are you sure it is fixed permanently?

the only reason why I changed it is because it did not really work right. also that’s not how I fixed. I fixed by pinging the scratch websocket because before there was no pinging so scratch server thought the client was no longer connected so scratch closes the connection.